tools/classes.js

```diff
@@ -5,6 +5,7 @@
  */
 
 var classes = [
+  'foam.core.Action',
```

Comment:
I don't think we need this.  We're not using the actual foam.core.Action class on the java side anywhere.